index.php

```diff
@@ -666,7 +674,7 @@
                                 </label>
                                 <label>
                                     Останавливать раздачи с количеством пиров более:
-                                    <input id="forum-control-peers" class="myinput forum-props" type="text" size="10" title="Укажите числовое значение пиров, при котором требуется останавливать раздачи текущего подраздела в торрент-клиентах. Либо оставьте это поле пустым, чтобы использовать глобальное значение для регулировки раздач. Значение равное -1 вовсе исключит подраздел из регулировки" />
+                                    <input id="forum-control-peers" class="myinput forum-props spinner-peers" type="text" size="10" title="Числовое значение пиров, при котором требуется останавливать раздачи текущего торрент-клиента. Значение равное -1 исключит торрент-клиент из регулировки. См. подраздел 'Настройки управления раздачами.'" />
```

Comment:
It turned out that `spinner-peers` does not work well with the "Save settings" button press, for some reason. When you change a value in one of such fields and right after press the save button, the previous value is being sent to PHP. You have to click save once again to get the value be saved.

It is a pretty strange and since we are in the alpha development, I don't feel it should block the PR from being merged, but I wish we have fix for this issue as soon as possible after.